components/match2/wikis/rocketleague/player_ext_custom.lua

```diff
@@ -0,0 +1,100 @@
+---
+-- @Liquipedia
+-- wiki=rocketleague
+-- page=Module:Player/Ext/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local PageVariableNamespace = require('Module:PageVariableNamespace')
+local Table = require('Module:Table')
+local TeamTemplate = require('Module:TeamTemplate')
+
+local playerVars = PageVariableNamespace({namespace = 'Player', cached = true})
+
+local PlayerExt = Lua.import('Module:Player/Ext', {requireDevIfEnabled = true})
+
+local CustomPlayerExt = Table.deepCopy(PlayerExt)
+
+--- Asks LPDB for the team a player belonged to on a page. For specific uses only.
+---@param resolvedPageName string
+---@date resolvedPageName string
+function CustomPlayerExt.fetchTeamHistoryEntry(resolvedPageName, date)
+	if Logic.isEmpty(resolvedPageName) then
+		return
+	end
+
+	local conditions = {
+		'[[type::Notable]]',
+		'[[pagename::' .. mw.title.getCurrentTitle().text:gsub(' ', '_') .. ']]',
+		'[[name::' .. resolvedPageName .. ']]',
+	}
+	local datapoint = mw.ext.LiquipediaDB.lpdb('datapoint', {
+		limit = 1,
+		conditions = table.concat(conditions, ' AND '),
+		query = 'information',
+	})[1]
+
+	if datapoint and Logic.isNotEmpty(datapoint.information) then
+		return {
+			joinDate = date,
+			leaveDate = date,
+			template = datapoint.information:lower(),
+		}
+	end
+end
+
+--- Fills in the team of the player on the specified date, if it is not specified in the arguments.
+---@pageName string
+---@template string?
+---@options {date: string?, fetchPlayer: boolean?, savePageVar: boolean?, useTimeless: boolean?}
+function CustomPlayerExt.syncTeam(pageName, template, options)
```

Comment:
This function seems identical to the one it's inheriting from. Seems redundant

Reply:
Check the previous review above